### PR TITLE
Rating backend

### PR DESCRIPTION
**Pull request by AntonisTerzo**
In this pr i  fixed that we get data from cmsAdapter and also that we check if the user that left the review is verified.We only count the verified reviews.
And Conny did: added renderRatings.js to get the ratings rendered from the frontend script.

### CODE REVIEW

**Review by AlbinNils98 — DISMISSED** [review]
I think the code looks much cleaner now when including cmsadapter and only having one function for rating in reviewsutils. I can't find any issues with the code so i think you should merge! **Nice job!**

**Review by MadeleineDahlqvist — APPROVED** [review]
Looks good!

**Review by AlbinNils98 — APPROVED** [review]
Looks good. Now the frontend works with the servers routes and logic.

File: utils/reviewsUtils.js [hunk 1]
```diff
@@ -48,28 +46,22 @@ export async function postReview(cmsAdapter, review) {
   await cmsAdapter.postReview(review);
 }
 
-async function getMovieReview(id) {
-  const res = await fetch(`${API_BASE}/reviews?filters[movie]=${id}`);
-  const payload = await res.json();
-  const modifiedArr = payload.data.map((obj) => ({
+export async function getAverageRating(adapter, id) {
+  const payload = await adapter.loadMoviesReviews(id);
```
Comment (richardolsson) [inline utils/reviewsUtils.js:50]: Great use of the adapter!

File: utils/reviewsUtils.js [hunk 2]
```diff
@@ -48,28 +46,22 @@ export async function postReview(cmsAdapter, review) {
...
     id: obj.id,
     ...obj.attributes,
   }));
-  return modifiedArr;
-}
-
-export async function getAverageRating(id) {
-  const reviewsList = await getMovieReview(id);
+  let filteredReviews = filterVerified(modifiedArr);
   const imdbRes = await imdbRating(id);
```
Comment (richardolsson) [inline utils/reviewsUtils.js:56]: This can make unnecessary requests to IMDB. Instead of loading the IMDB rating here, it would be better to only load it if the number of filtered reviews is less than 5. If there are more than five verified reviews, IMDB rating will still be loaded here, and then not used anywhere.

File: utils/reviewsUtils.js [hunk 3]
```diff
@@ -79,6 +71,5 @@ export async function getAverageRating(id) {
     averageRating = imdbRes;
     maxRating = 10;
   }
-
```
Comment (richardolsson) [inline utils/reviewsUtils.js:82]: This change just removes an empty line of code. Avoid making such unnecessary changes in order to keep PRs clean and easier to review.

**Review by richardolsson — COMMENTED** [review]
Nice work! I've left some comments on the backend code here, but many of the changes weren't really substantial changes to backend logic, so there was not much to comment on for this particular PR. I hope it can still be useful!
